rosserial_client/CMakeLists.txt

```diff
@@ -10,9 +10,10 @@ catkin_python_setup()
 
 if(CATKIN_ENABLE_TESTING)
   find_package(rosserial_msgs REQUIRED)
-  include_directories(src ${rosserial_msgs_INCLUDE_DIRS})
+  include_directories(src/ros_lib ${rosserial_msgs_INCLUDE_DIRS})
```

Comment:
I think this is fine for now, but given the potential for weird header cross talk issues here between tests, we should probably move this over to use `target_include_directories` on a per-test basis to ensure that each one gets just what it needs.